docs/post_installation/peering.md

```diff
@@ -47,69 +37,65 @@ Your p2p public key (base58):  8CZELJwB3aBzxJgnLMvvt1FirAwNN6jif9LavYTNHCty
 Your p2p PeerID:  12D3KooWH1vQ5SWtEUTVNCCCxxgGeoLZLVz1PnpCcjhcJXcGB9cu
 ```
 
-Now simply copy the value of `Your p2p private key` to the `p2p.identityPrivateKey` configuration option.
+Now, copy the value of `Your p2p private key` to the `p2p.identityPrivateKey` configuration option.
 
 Your Hornet node will now use the specified private key in `p2p.identityPrivateKey` to generate the `PeerId` (which will
 ultimately be stored in `./p2pstore`).
 
-> In case there already is a `./p2pstore` with another identity, Hornet will panic and tell you that you have a previous identity which does not match with what is defined via `p2p.identityPrivateKey` (
-in that case either delete the `./p2pstore` or reset the `p2p.identityPrivateKey`).
+:::info
+In case there already is a `./p2pstore` with another identity, Hornet will panic and tell you that you have a previous identity which does not match with what is defined via `p2p.identityPrivateKey` .  In this case, you should either delete the `./p2pstore` or reset the `p2p.identityPrivateKey`.
```

Comment:
```suggestion
In case there already is a `./p2pstore` with another identity, Hornet will alert you that you have a previous identity which does not match with what is defined via `p2p.identityPrivateKey` .  In this case, you should either delete the `./p2pstore` or reset the `p2p.identityPrivateKey`.
```